font-stix-otf.yaml

```diff
@@ -1,44 +1,31 @@
-# cffsubr has issues when building on aarch64
+# 2.13b171 is the latest version in github releases: https://github.com/stipub/stixfonts/releases
 package:
   name: font-stix-otf
-  version: 2.14
+  version: 2.13.0
   epoch: 1
   description: stix-otf fonts
   copyright:
     - license: OFL-1.1
-  target-architecture:
-    - x86_64
 
 environment:
   contents:
     packages:
       - build-base
       - busybox
-      - cmake-3
-      - libxml2-dev
-      - py3-pip
-      - python-3.12
-      - python-3.12-dev
 
 pipeline:
-  - uses: git-checkout
+  - uses: fetch
     with:
-      repository: https://github.com/stipub/stixfonts
-      tag: v${{package.version}}
-      expected-commit: de537a95240e0105546bed58e15b4a61be28cd89
+      uri: https://github.com/stipub/stixfonts/archive/refs/tags/v2.13b171.tar.gz
+      expected-sha256: 1e76b9ab0bb08372ff73ad5b58d9116260e9058d1fce4b83fe1e213c3b9c947f
 
   - runs: |
-      python3 -m venv venv
-      source venv/bin/activate
-      pip3 install -r requirements.txt
-      python tools/tirobuild.py STIXbuild.yml
       mkdir -p "${{targets.contextdir}}/usr/share/fonts/stix-fonts/"
-      mv ./output/STIXbuild/STIXTwoMath/OTF/* "${{targets.contextdir}}/usr/share/fonts/stix-fonts/"
-      mv ./output/STIXbuild/STIXTwoText/OTF/* "${{targets.contextdir}}/usr/share/fonts/stix-fonts/"
+      mv ./fonts/static_otf/ "${{targets.contextdir}}/usr/share/fonts/stix-fonts/"
 
 update:
   enabled: false
```

Comment:
Can we set the `enabled: true` and pass `schedule:`?

Reply:
tried a different approach here!